src/Aggregator.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Algolia\SearchBundle;
+
+use Algolia\SearchBundle\Exception\EntityNotFoundInObjectID;
+use Algolia\SearchBundle\Exception\InvalidEntityForAggregator;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Serializer\Normalizer\NormalizableInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+abstract class Aggregator implements NormalizableInterface
+{
+    /**
+     * Holds the ObjectID.
+     *
+     * Typically also contains information concerning the
+     * entity class name, and concerning the entity id.
+     *
+     * @ORM\Id
```

Comment:
@nunomaduro these annotations can be removed as well: they won't be considered since the class itself is not mapped.

Reply:
@alcaeus I would like to leave the annotation here, otherwise developers will have to define one on the children class:
```
  [Doctrine\ORM\Mapping\MappingException]
  No identifier/primary key specified for Entity "App\Entity\News" sub class
  of "Algolia\SearchBundle\Aggregator". Every Entity must have an identifier/
  primary key.
```

Reply:
Fixed!